src/Field/Image.php

```diff
@@ -60,11 +60,12 @@ public function process($field)
     {
         $attachmentId = $this->fetchValue($field);
 
-        $attachment = $this->post->attachment()->find(intval($attachmentId));
+        $attachment = Post::find(intval($attachmentId));
```

Comment:
when using `Post::find()` you'll get a new query using `newQuery()` so you will not get the current database connection that came with the post you're calling, like `$post->acf->...`, because I get the connection from the post object...

I'm gonna try it in a different way and let you know :-)

thanks for helping!